Jenkinsfile

```diff
@@ -29,6 +30,77 @@ pipeline {
                 sh 'build/run make -j\$(nproc) test-integration'
             }
         }
+
+        stage('Integration Tests SetUp') {
+            steps {
+                sh 'tests/scripts/saveImages.sh > BUILD_NUMBER'
```

Comment:
why is the BUILD_NUMBER needed? can you not tag the image to something stable in saveImages.sh?